src/libstd/f32.rs

```diff
@@ -188,7 +188,7 @@ impl f32 {
         if self.is_nan() {
             NAN
         } else {
-            unsafe { intrinsics::copysignf32(1.0, self) }
```

Comment:
Note that `copysign` is a method so `1.0_f32.copy_sign(self)` would work.

Reply:
I fixed code following your comment.